src/main/java/com/wolox/training/controllers/BookController.java

```diff
@@ -51,6 +46,11 @@ public String greeting(@RequestParam(name="name", required=false, defaultValue="
      * @return the book with the id passed as parameter
      */
     @GetMapping("/{id}")
+    @ApiOperation(value = "Giving an Id, return the book", response = Book.class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Successfully retrieved book"),
+            @ApiResponse(code = 404, message = "Book not found")
+    })
     public Book findOne(@PathVariable Long id) throws NotFoundException {
```

Comment:
You could document the param with the @ApiParam annotation informing the value it should contain and if it is required or not.